pkg/infra/spot.go

```
@@ -259,40 +263,35 @@ var _ = Describe("[sig-cluster-lifecycle] Machine API Running on Spot", framewor
-				Expect(err).ToNot(HaveOccurred(), "Should be able to get Machines from MachineSet")
-				Expect(len(machines)).To(BeNumerically(">", 0), "There should be at least one Machine")
+			machines, err := framework.GetMachinesFromMachineSet(ctx, client, machineSet)
+			Expect(err).ToNot(HaveOccurred(), "Should be able to get Machines from MachineSet")
+			Expect(len(machines)).To(BeNumerically(">", 0), "There should be at least one Machine")
 
-				customRand := rand.New(rand.NewSource(time.Now().Unix()))
-				machine = machines[customRand.Intn(len(machines))]
-				Expect(machine.Status.NodeRef).ToNot(BeNil(), "Machine should have a linked Node")
-			})
+			for _, machine := range machines {
 
```

golangci-lint (2.12.2)

[error] unnecessary whitespace (leading-whitespace)

(wsl_v5)